### PR TITLE
expose responder method for testing and overriding globally

### PR DESCRIPTION
**Pull request by bobholt**

### CODE REVIEW

File: modules/controller/index.js [hunk 1]
```diff
@@ -60,7 +61,7 @@ Controller.prototype.create = function (opts) {
       var payload = createResponse(err, data, _.extend({}, opts, {
         type: type
       }));
-      responder(payload, request, response);
+      this.responder(payload, request, response);
```
Comment (tkellen) [inline modules/controller/index.js]: the context of `this` here will be wrong, maybe to `var responder = this.responder` outside the closure like the rest?

Comment (bobholt) [inline modules/controller/index.js]: Oops. Yep.

File: modules/controller/index.js [hunk 2]
```diff
@@ -108,7 +110,7 @@ Controller.prototype.update = function (opts) {
   return function (request, response) {
     source.byId(request.params.id, function (err, model) {
       if (!model) {
-        return responder(lookupFailed, request, response);
+        return this.responder(lookupFailed, request, response);
```
Comment (tkellen) [inline modules/controller/index.js]: Shouldn't this be `respond(....`?

File: modules/controller/index.js [hunk 3]
```diff
@@ -142,7 +144,7 @@ Controller.prototype.destroy = function (opts) {
...
```
Comment (tkellen) [inline modules/controller/index.js]: Shouldn't this be `respond(....`?

File: modules/controller/index.js [hunk 4]
```diff
@@ -151,7 +153,7 @@ Controller.prototype.destroy = function (opts) {
           var payload = destroyResponse(err, data, _.extend({}, opts, {
             type: type
           }));
-          responder(payload, request, response);
+          this.responder(payload, request, response);
```
Comment (tkellen) [inline modules/controller/index.js]: Shouldn't this be `respond(....`?

**Pull request merged**